### PR TITLE
Feature/infection chart

### PR DESCRIPTION
**Pull request by Ulydev**
This PR adds a visualisation of the probability of infection through a chart.

### CODE REVIEW

**Comment by Ulydev** [comment]
Feature is ready
Screenshot_1588863049

File: app/src/main/java/ch/epfl/sdp/TabActivity.java [hunk 1]
```diff
@@ -31,7 +32,7 @@
 
     private static final Tab[] tabs = new Tab[]{
             new Tab(MapFragment.class, R.string.tab_map, R.drawable.tab_map),
-            new Tab(UserInfectionFragment.class, R.string.tab_status, R.drawable.tab_status),
+            new Tab(StatusFragment.class, R.string.tab_status, R.drawable.tab_status),
```
Comment (AdrienRem) [inline app/src/main/java/ch/epfl/sdp/TabActivity.java:35]: Good idea to rename that class !

**Review by AdrienRem — APPROVED** [review]
I don't see tests but it's hard to test graphical components so I accept this. If we find a way to do it I'm all for it

**Review by matteopariset — APPROVED** [review]
Very good-looking graph

**Pull request merged**